Specs/Data/Tiles3D/instanced/instancedEmptyGltf/tiles.json

```diff
@@ -0,0 +1 @@
+{"properties":{"id":{"minimum":0,"maximum":99},"height":{"minimum":0.7289306027814746,"maximum":97.26047022268176}},"geometricError":40,"root":{"box":[-1.319970476767974,0.6986942109000001,-1.3199295232320258,0.6987257891,0,15],"geometricError":0,"content":{"url":"instancedEmptyGltf.i3dm"}}}
```

Comment:
Can we use human-readable formatting for all tiles.json examples.  There should be a command-line option on our tool.  For the existing content, I'm sure Sublime or Web Storm can do it, or use https://jsonformatter.curiousconcept.com/